scripts/process-integration-results.sh

```diff
@@ -11,6 +11,10 @@ RESULTSDIR=${RESULTSDIR-"$NETWORK_NAME/results"}
 
 [ $# -gt 0 ] && RESULTSDIR="$1"
 
+clean_slog() {
+  jq -cr 'del(.time, .monotime, .dr[2].timestamps, .memoryUsage, .heapStats, .statsTime)'
```

Comment:
be on the lookout for some `kernelStats` that @FUDCo will be adding to one of the cosmic-swingset `end-block` slog records, maybe next week, I think some of them are non-consensus

Reply:
Yeah but I believe all of them are actually identical. Regardless they are already written to the swingstore and there is no diff showing up so far ;)